libraries/botbuilder-vendors/vendors/recognizers-text/recognizers-text/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "@microsoft/recognizers-text",
+  "version": "1.1.4",
+  "main": "index.js",
+  "dependencies": {
+		"xregexp": "^4.2.0"
```

Comment:
Is this indentation correct?

Reply:
Locally it looks fine, but when i formatted it something changed, i think it is related to the tab formatting using a different hidden character, thanks!.